workspace/spago.dhall

```diff
@@ -0,0 +1,21 @@
+{ name = "cw-purescript"
+, dependencies =
```

Comment:
Is it possible to silence `spago`?

User will see warnings like this:
```text
 [warn] None of your project files import modules from some projects that are in the direct dependencies of your project.
These dependencies are unused. To fix this warning, remove the following packages from the list of dependencies in your config:
- bigints
- console
- debug
- profunctor-lenses
- quickcheck
- rationals
- spec-quickcheck
```

Reply:
I'm not sure, I think there is no way to suppress these warnings. But I think they won't show up multiple times. It could work to separate `spago install`, `spago build --no-install` and maybe `spago run`.

Are all logs shown to the user as they are? Or is there a way to filter some of them out or start collecting logs after installation?

Reply:
Oh, and a hacky solution would be to create a file that imports all of these packages, so that they are in use (e.g. test)

Reply:
It's shown when you do `spago test`. See https://github.com/codewars/purescript/pull/1/checks?notification_referrer_id=NT_kwDOAAnBaLE0MzU4OTQ5NzA4OjYzOTMzNg#step:5:8

> Are all logs shown to the user as they are? Or is there a way to filter some of them out or start collecting logs after installation?

Logs in that workflow step are shown to the user unless we do something. The test outputs are parsed and rendered as you see on Codewars. I can post process the output to remove those warnings, but I wanted to know if there's a better way.

I wouldn't want to create and maintain a file that imports all of them.

Reply:
> I wouldn't want to create and maintain a file that imports all of them.

Understandable 😉

> Logs in that workflow step are shown to the user unless we do something. 

To clarify: Are we talking about the user as somebody who looks into the github actions logs (then I'd ignore it or `echo` that these are expected in the workflow) or are we talking about the codewars user trying to solve a kata (because right now I can only see some output like tests and stderr)?

Reply:
![image](https://user-images.githubusercontent.com/13085980/189894645-220ac76c-0845-4ca7-853a-5b57bcd0dcfa.png)

Reply:
> To clarify: Are we talking about the user as somebody who looks into the github actions logs (then I'd ignore it or echo that these are expected in the workflow) or are we talking about the codewars user trying to solve a kata (because right now I can only see some output like tests and stderr)?

Trying to solve a kata. The current output with `spago test` will look like the following on Codewars:

![image](https://user-images.githubusercontent.com/639336/190026200-3ebb8a1c-527b-4320-b7dd-08350b5cea4b.png)

I can postprocess the output if it's not supported.

Reply:
I forgot we already postprocess `pulp test`, so I'll do something similar.

Reply:
Other than this the PR seems to look good now :)